Tribler/Core/Modules/restapi/display_endpoint.py

```diff
@@ -0,0 +1,89 @@
+import json
+
+from twisted.web import http, resource
+
+
+class DisplayEndpoint(resource.Resource):
+    """
+    This endpoint is responsible for handling requests for the trust display.
+    """
+
+    def __init__(self, session):
+        """
+        Initialize the DisplayEndpoint
```

Comment:
Separate the description and the argument docs with a newline. Also, end with a `.`.